aquality-appium-mobile-template-cucumber/src/test/java/aquality/appium/mobile/template/cucumber/objectfactory/ServiceModule.java

```diff
@@ -0,0 +1,15 @@
+package aquality.appium.mobile.template.cucumber.objectfactory;
+
+import aquality.appium.mobile.application.AqualityServices;
+import aquality.appium.mobile.screens.screenfactory.IScreenFactory;
+import aquality.appium.mobile.template.utilities.ScreenshotProvider;
+import com.google.inject.AbstractModule;
+
+public class ServiceModule extends AbstractModule {
+
+    @Override
+    protected void configure() {
+        bind(IScreenFactory.class).to(AqualityServices.getScreenFactory().getClass());
```

Comment:
probably we should bind to instance here

Reply:
Why? AqualityServices.getScreenFactory() - it is an instance. We have to bind to instance when we don't have interfaces, isn't it?